docs/blog/august-contributions-showcase.md

```diff
@@ -0,0 +1,134 @@
+---
+title: August contributions showcase
+author:
+  - name: James Garbutt
+sidebar: false
+date: 2024-09-07
+head:
+  - - meta
+    - property: og:type
+      content: website
+  - - meta
+    - property: og:title
+      content: August contributions showcase
+  - - meta
+    - property: og:url
+      content: https://e18e.dev/blog/august-contributions-showcase
+  - - meta
+    - property: og:description
+      content: A showcase of e18e-focused contributions from August
```

Comment:
an og image would help for sharing on twitter, etc.

Reply:
For sure. I'm not very good with images so any help would be appreciated 😅